locksmith/src/controllers/userController.ts

```diff
@@ -83,6 +83,22 @@ namespace UserController {
       return res.sendStatus(400)
     }
   }
+
+  export const updatePaymentDetails = async (
+    req: Request,
+    res: Response
+  ): Promise<any> => {
+    let emailAddress = req.params.emailAddress
+    let token = req.body.token
+
+    let result = await UserOperations.updatePaymentDetails(token, emailAddress)
```

Comment:
IIRC we decided to not use emailAddress but `publicKey` as the universal account identifier?

Reply:
In the data it absolutely is, i'll swap this endpoint to use that as the lookup key. The general use case for this endpoint a calling application generates a strike token and passes it forward with the  information on hand.